data/campaigns/The_Rise_Of_Wesnoth/scenarios/03_A_Harrowing_Escape.cfg

```diff
@@ -145,6 +145,66 @@
         [/message]
     [/event]
 
+    [event]
+        name=turn 2
+        [filter_condition]
+            [not]
+                [have_unit]
+                    id=Burin the Lost
+                [/have_unit]
+            [/not]
+        [/filter_condition]
+
+        [move_unit_fake]
+            type=Dwarvish Steelclad
+            x=8,6
+            y=5,4
+        [/move_unit_fake]
+        [unit]
+            id=Burin Hint
+            name= _ "Unknown Dwarf"
```

Comment:
Please make this `_ "Burin the Lost"`, so that the translators don't get an extra string to translate.

Reply:
@stevecotton done